### PR TITLE
feat(api): add notices endpoint

### PR DESCRIPTION
**Pull request by JalilArfaoui**
Add `/notices` endpoint to get the list of all publicly visible notices.

Can be filtered with get parameter `?contributor=42`

Comparing to existing `/notices/{id}` :
- replaced `contributor` (with details) by `contributorId` (id only)
- removed `intention` in response
- add `exampleUrl`

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -105,7 +105,7 @@ While the prerequisites above must be satisfied prior to having your pull reques
 
 ### PHP Styleguide
 
-All JavaScript must adhere to [PHP The Right Way](https://phptherightway.com/#code_style_guide) code style.
+All JavaScript must adhere to [PHP The Right Way](https://phptherightway.com/#code_style_guide) code style.
```
Comment (lutangar) [inline CONTRIBUTING.md:108]: Now I know that you miss JavaScript <3

Comment (JalilArfaoui) [reply CONTRIBUTING.md:108]: It was a non-breakable space, that I happen to type sometimes on my Bépo keyboard :-)

Comment (lutangar) [reply CONTRIBUTING.md:108]: > It was a non-breakable space

:sweat_smile: in fact didn't notice that

I was talking about the fact you are speaking about Javascript in the PHP Styleguide section and point to a PHP coding style. Or maybe there are rules about JavaScript in this styleguide I didn't know about?!


All JavaScript must adhere to [PHP The Right Way](https://phptherightway.com/#code_style_guide) code style.	All JavaScript must adhere to [PHP The Right Way]

Comment (JalilArfaoui) [reply CONTRIBUTING.md:108]: LOL ! I didn’t even notice my mistake, despite your comment !

**Pull request merged**